e2e-tests/web/wdio.conf.ts

```diff
@@ -22,27 +23,29 @@ export const config: WebdriverIO.Config = {
   logLevel: 'info',
   bail: 0,
   baseUrl: 'http://localhost:9000',
-  waitforTimeout: 2000,
-  connectionRetryTimeout: 120000,
+  waitforTimeout: 2_000,
+  connectionRetryTimeout: 120_000,
   connectionRetryCount: 3,
   services: process.env.CI ? [] : ['selenium-standalone'],
   framework: 'jasmine',
   reporters: ['spec'],
 
   jasmineOpts: {
-    defaultTimeoutInterval: 300000,
+    defaultTimeoutInterval: 300_000,
   },
 
   onPrepare: async (): Promise<void> => {
     if (process.env.CI) {
-      const startupDelay = 20000
+      const startupDelay = 10_000
```

Comment:
a delay is still required for web pack to actually start. otherwise the tests will fail with econrefused.